ar/publish.css

```diff
@@ -10,15 +10,16 @@ pre {
 	padding: 32px 18px 0 0;
 }
 .site-body-left-column-site-name {
-	padding: 4px 0 32px 32px;
+	/* padding: 4px 0 32px 32px; */
+	display: none;
 }
 .site-body-left-column .site-body-left-column-inner {
 	margin-left: initial;
 	margin-inline-start: auto;
 }
 .site-body-left-column .search-view-outer {
 	padding-right: initial;
-	padding-inline-end: 18px;
+	/* padding-inline-end: 18px; */
```

Comment:
Do you still need this? Would prefer to delete commented out code if no longer needed

Reply:
No, I don't need it.